Editor/EdgegapWindowV2.cs

```diff
@@ -796,6 +809,7 @@ private async Task verifyApiTokenGetRegistryCreds()
             // Disable most ui while we verify
             _isApiTokenVerified = false;
             _signOutBtn.SetEnabled(false);
+            _apiTokenVerifyBtn.SetEnabled(false);
```

Comment:
let's avoid disabling UI, if the code fails anywhere inside the function the user will be stuck in a partial error state. since we have the early exit pattern and don't run verification if it's already running, disabling the input doesn't add much value.